sound/soc/amd/acp/Kconfig

```diff
@@ -112,6 +112,9 @@ config SND_SOC_ACPI_AMD_MATCH
 	tristate
 	select SND_SOC_ACPI if ACPI
 
+config SND_SOC_AMD_SOF_BOARD_HELPERS
+	tristate
+
```

Comment:
nit-pick: the term 'monolithic' is generally used for an all-in-one device. A monolithic card is not really a thing, even less so when it's made of 3 or 4 discrete component.

This setup is generally referred to as an AIC (Add-In-Card) or AOC (Add-On-Card). It's certainly not monolithic in terms for functionality with lots of jumpers and double-digit number of configurations.

Reply:
will fix it.